src/Api/Model/Languageforge/Lexicon/Command/LexUploadCommands.php

```diff
@@ -101,8 +101,8 @@ public static function uploadAudioFile($projectId, $tmpFilePath)
 
                 if (
                     strcmp($project->whenToConvertAudio, "always") == 0 ||
-                    (strcmp($project->whenToConvertAudio, "sr") == 0 && // restrictions on send/receive files 12-2022
-                        (filesize($filePath) > 1000000 ||
+                    (strcmp($project->whenToConvertAudio, "sr") == 0 && // restrictions on send/receive files 01-2023
+                        (filesize($filePath) > 10000000 ||
```

Comment:
Where we display warnings for files that are too big, I added links to the Chorus code that defines this limit. That seemed like the most helpful way to explain the magic number. E.g. like this.
```suggestion
                    (strcmp($project->whenToConvertAudio, "sr") == 0 &&
                        // `MaximumFileSize` in https://github.com/sillsdev/chorus/blob/master/src/LibChorus/FileTypeHandlers/audio/AudioFileTypeHandler.cs
                        (filesize($filePath) > 10000000 ||
```
What do you think?